.github/workflows/fast-forward.yml

```diff
@@ -18,7 +18,7 @@ jobs:
           require: 'write'
 
       - name: Comment PR
-        uses: thollander/actions-comment-pull-request@v2
+        uses: thollander/actions-comment-pull-request@v3
         if: steps.has_permission.outputs.require-result != 'true'
         with:
           pr_number: ${{ github.event.issue.number }}
```

Comment:
according to https://github.com/thollander/actions-comment-pull-request/releases/tag/v3.0.0
we should rename these parameters:

- From pr_number to pr-number
- From comment_tag to comment-tag
- From create_if_not_exists to create-if-not-exists